eva-accession-clustering-automation/cluster_multiple_assemblies.py

```diff
@@ -44,6 +47,10 @@ def generate_bsub_command(assembly_accession, properties_path, clustering_artifa
     return command
 
 
+def get_job_name(assembly_accession):
+    return 'cluster_{assembly_accession}'.format(assembly_accession=assembly_accession)
```

Comment:
we have the option here of putting a timestamp in the job name. That way, if a run fails, the next run will be using a different set of dependencies.

In the past we have had problems when submitting different jobs with the same name, if they were submitted too fast (right after the previous one failed).

Reply:
Good point, I just changed it to include the timestamp in the job name